### ISSUE TITLE
Format of test data specid-vendordata.bin

### ISSUE THREAD

**Issue by JuergenReppSIT** [tpm2-software/tpm2-tools#2190]
I did use this file for tests with tpm2-tss fapi and noticed that the size (EventSize) of the event data did not include VendorInfo. VendorInfo was displayed by tpm2_eventlog but the size was not checked.

**Comment by idesai** [comment]
@JuergenReppSIT any update on this or can we close it?

**Comment by JuergenReppSIT** [comment]
@idesai Yes it can be closed. If in the future FAPI event logging will be integrated we can add the file which I currently use in the FAPI tss2 tests.
